cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -182,7 +182,7 @@ describe('Script Builder', () => {
       cy.wait('@query -15m')
     })
 
-    describe('data completeness', () => {
+    describe.skip('data completeness', () => {
```

Comment:
This will fail because:
(1)  the download now has a dynamically downloaded csv name
   * so we need to i[ntercept the request response](https://github.com/influxdata/ui/blob/master/cypress/e2e/shared/explorer.test.ts#L634-L646), instead of presuming an exact name.

(2) since a form submit, we need to mock the page reload.
   * cypress will fail tests unless this happens.
   * it's done [this way](https://github.com/influxdata/ui/blob/master/cypress/e2e/shared/explorer.test.ts#L621-L623), (as pulled directly from cypress docs)

Reply:
Thanks for looking into this! I updated the test